rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/ClientProxyImpl.java

```diff
@@ -119,7 +119,17 @@ public ClientProxyImpl(URI baseURI,
                            boolean isRoot,
                            boolean inheritHeaders,
                            Object... varValues) {
-        this(new LocalClientState(baseURI), loader, cri, isRoot, inheritHeaders, varValues);
+        this(baseURI, loader, cri, isRoot, inheritHeaders, Collections.emptyMap(), varValues);
```

Comment:
Thanks a lot for review, @andymc12! The `Collections.emptyMap()` is actually backed by the constant, basically the only purpose of the method is to cast the generic parameters (this is implementation from `Collections`):
```
    public static final <K,V> Map<K,V> emptyMap() {
        return (Map<K,V>) EMPTY_MAP;
    }
```

Using `null` is also possible, but is a bit risky since `ClientProxyImpl` has a number of overloaded constructors.

Sounds good? Thanks a lot!

Reply:
That makes sense, thanks.